### PR TITLE
prepare and release version 1.2.0

### PR DESCRIPTION
**Pull request by vsamel**
- fix unsafe handling of printf() args
- remove RELEASE-NOTES file
- update doc after rvnamed-ng merge
- remove iptraf-ng.spec file
- update CHANGES file
- release 1.2.0

### CODE REVIEW

**Comment by pecameron** [comment]
@vsamel  This looks good to me.
/lgtm
We are overdue for a new release and this seems like a good time.
When we make the release we need to create a new release using the release button. This will create tarballs for the source that can be referenced by fedora builds and other release software.

File: CHANGES [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
-* Philip Cameron <pcameron@redhat.com> - 1.1.5
-- Incorporate numerous cleanup changes in the code
-- Revised documentation. Previous distributions shipped the
-  original IPTraf documentation.
-- Bugfix: segfault in rate_add_rate
-- Bugfix: Fix memory leak when destroying iflist
+* Mon Jun 01 2020 Vitezslav Samel <vitezslav@samel.cz> - 1.2.0
+- ipfrag: code refactoring (Nikola Pajkovsky)
+- ifstats: sort interfaces by name (Jan Engelhardt)
+- ifstats: fix memory leak (Phil Cameron)
```
Comment (pecameron) [inline CHANGES:6]: @vsamel 
I fixed issue 23 and it needs to be in release 1.2
ipfilter - fix display of internal memory (issue 23) (Phil Cameron)
Please add above or similar.

**Comment by pecameron** [comment]
@vsamel the pr23 bug fix needs to be mentioned in changelog and PR comment above since it is a response to a customer reported bug. The bugzilla reference should be included whenever we explicitly fix a reported bug. I think we now support infiniband (if so should be mentioned). I am tracking the following, you may know of more to add.

bugzilla 1842690 - iptraf exposes raw memory contents which can result in privilege elevation or data theft/tampering.
bugzilla 1572750 - iptraf-ng hogs the cpu if the interface gets removed
bugzilla 1140211 -  [RFE] IPoIB: iptraf-ng not showing IPoIB traffic

Thanks for everything you have been doing to improve iptraf-ng.

**Comment by vsamel** [comment]
I've fixed the CHANGES file to include RedHat Bugizlla numbers.

**Comment by vsamel** [comment]
@pecameron Are there any other problems with this PR?

**Comment by pecameron** [comment]
@vsamel is the data leak problem fixed? I closed my version for a fix from you.

**Comment by vsamel** [comment]
@pecameron It's fixed by the first hunk of 4fe64c0 patch in this PR. So yes, it's fixed.

**Comment by pecameron** [comment]
@vsamel I made the release:
https://github.com/iptraf-ng/iptraf-ng/releases/tag/v1.2.0
Thanks for all the work you have done to make this possible.

**Pull request merged**